contracts/ainft/AINFTBaseV2.sol

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: UNLICENSED
+// AINFT Contracts v1.0.0
```

Comment:
v2?

Reply:
Sorry, I made a mistake 😭
It is specified as v2, but it is a minor change compared to v1. 
I'm wondering if it's better to specify it as v2 or v1.0.1.

Reply:
If it's just minor, how about updating the existing files rather than adding new ones?

Reply:
sorry, I made a mistake. I'll fix it with v2.0.0.

Reply:
I added an override implementation to v2. 
If I modify v1, I am worried that there will be compatibility issues with other contracts that are already using v1.
Can I modify the contract because it is deployed only once to Ethereum?

Reply:
I guess no one is actually using this base contract now as we've just used v1 once for an internal workshop in Q4. Even if there are people using it, they can control the compatibility by the npm package version?

Reply:
Yes, the latest version of [@ainft-team/ainft-contracts](https://www.npmjs.com/package/@ainft-team/ainft-contracts) is 1.0.1. I think it would be okay to modify v1 and release it as 1.0.2, I'll try that!
Thanks for the helpful advice 😊

Reply:
May I merge this branch and create a new branch integrating v1 and v2? I will specify the contract version as 1.0.2.

Reply:
Looks good to me. I think we can go with v1.1.0 once integrated.